semperfiwebdesign/all-in-one-seo-pack
# Changed Support Metabox Icons to Font-Icons

Issue #289 #1154 

## Proposed changes

Changes the icons in Support sidebar metabox to font icons.

## Types of changes

- New feature (non-breaking change which adds functionality)
- Improves existing functionality
- Improves existing code

## Checklist


## Testing instructions

1. Activate plugin.
2. Go to AIOSEOP General Settings.
3. Scroll to Support sidebar metabox, and icons should be font icons.

## Further comments

![support metabox](https://user-images.githubusercontent.com/2794445/58363731-dc9f5a80-7e5d-11e9-991b-cfcb692c0c3a.png)
